README.md

```diff
@@ -320,7 +332,7 @@ which is implemented in `mapget::Service`. Detailed endpoint descriptions:
 | `/abort`   | POST   | Abort a currently running `/tiles` request by its `clientId`.                                                     | `clientId`                                                                                                                                          | `text/plain`                                                                                                                                                                                                                                                      |
 | `/status`  | GET    | Server status page                                                                                                | None                                                                                                                                                | `text/html`                                                                                                                                                                                                                                                       |
 | `/locate`  | POST   | Obtain a list of tile-layer combinations providing a feature that satisfies given ID field constraints.           | `application/json`: List of external references, where each is a Request object with `mapId`, `typeId` and `featureId` (list of external ID parts). | `application/json`: List of lists of Resolution objects, where each corresponds to the Request object index. Each Resolution object includes `tileId`, `typeId`, and `featureId`.                                                                                 |
-| `/config`  | GET    | Access the config yaml-file content.                                                                              | None                                                                                                                                                | `application/json`: Contains the `sources` and `http-settings` from the config-yaml as a JSON representation. The returned JSON object has a `model`, `schema` and `readOnly` key. The schema is controlled through the `--config-schema` command line parameter. |
+| `/config`  | GET    | Access the config yaml-file content. Disabled iff `--no-get-config` is passed to mapget.                          | None                                                                                                                                                | `application/json`: Contains the `sources` and `http-settings` from the config-yaml as a JSON representation. The returned JSON object has a `model`, `schema` and `readOnly` key. The schema is controlled through the `--config-schema` command line parameter. |
```

Comment:
We need a corresponding erdblick change here to handle the GET /config 403 response.